types/public/Interfaces.d.ts

```diff
@@ -10,6 +10,12 @@
  * result will be passed to JSON.stringify unless it is a string, Buffer, ArrayBufferView, or number.
  */
 export declare type AzureFunction = ((context: Context, ...args: any[]) => Promise<any> | void);
+export interface ContextBindings {
+    [name: string]: any;
+}
+export interface ContextBindingsData {
```

Comment:
Nit: Maybe drop the "s" for `ContextBindingData`? 
Consistent with "bindings" => ContextBindings and "bindingData" => ContextBindingData

Reply:
Opps, good spotting! 😅